### DISCUSSION TITLE
Panel: Ensures aria-hidden elements are not focusable nor contain focusable elements

### DISCUSSION THREAD

**Discussion by Bryelmo** [primefaces/community#457] [PrimeNG] [answered]
It would be better for Panel component pass the accessibility WCAG 2.1 AA standard test. I'm using it in a project that requires this standard.

Panel component fails a WCAG 2.1 AA standard test with Axe Devtools and Accessibility insights for web extensions on Google Chrome browser.

`<div role="region" class="p-toggleable-content ng-tns-c697-106 ng-trigger ng-trigger-panelContent" id="p-panel-3-content" aria-hidden="true" aria-labelledby="p-panel-3-titlebar" style="height: 0px;">`

Axe Devtools says how to solve this problem, you need to fix at least (1) of the following:

- Focusable content should have tabindex="-1" or be removed from the DOM

**Comment by mehmetcetin01140** [comment, marked as answer]
Hi,

Thanks for the feedback, we've updated the component. Update will be available in next release.

**Reply by Bryelmo** [reply]
Thanks @mehmetcetin01140 for your job, I will continue to create accessibility issues for your improvements
